CONTRIBUTION.md

```diff
@@ -52,6 +62,8 @@ In general apply common sense to the Code you write. "Effective Java" is also al
 * Use Guava's `Preconditions` class for Argument validation
 * Validate `@NonNull` values with `Objects.requireNonNull` if necessary
 * use getters and setters, even in the declaring class
+* avoid getters and setters for protected methods
```

Comment:
```suggestion
* avoid getters and setters for protected fields
```
Also this needs clarification that in the API, the reverse holds true...

Reply:
Will review when we get an API?

Reply:
I'd rather have it in here already, as we have the beginning in 1.13 already and hn's PR target's API code so... There are already PR's targeting the API...

Reply:
```suggestion
* Avoid getters and setters for protected fields in non API-Code. 
* Avoid non-private variables in API-Code and provide getters/setters instead. Bear in mind that this does not apply to constants (`public static final type xyz = ...`) though.
```